### PR TITLE
CHE-1248 remove workspace information from EnvironmentContext

### PR DESCRIPTION
**Pull request by mshaposhnik**

### CODE REVIEW

File: wsagent/che-core-api-git/pom.xml [hunk 1]
```diff
@@ -87,6 +87,10 @@
             <artifactId>che-core-commons-lang</artifactId>
         </dependency>
         <dependency>
+            <groupId>org.eclipse.che.core</groupId>
+            <artifactId>wsagent-local</artifactId>
```
Comment (sleshchenko) [inline wsagent/che-core-api-git/pom.xml:91]: che-core-api-git will be used for codenvy application also. Are you sure that it is OK that wsagent-local will be used there?

**Comment by sleshchenko** [comment]
fix java doc here https://github.com/eclipse/che/pull/1658/files#diff-ce3c54a447aeb1135e810a8cfe16f30dR31

File: wsagent/che-core-api-git/pom.xml (hunk 1 above)
Comment (mshaposhnik) [inline wsagent/che-core-api-git/pom.xml:91]: Ok for now.

**Comment by skabashnyuk** [comment]
ok

**Comment by musienko-maxim** [comment]
After checking all tests we do not have any regressions as compared with nightly4 server.

**Pull request merged**